pkg/api/admin_users.go

```diff
@@ -4,12 +4,12 @@ import (
 	"github.com/grafana/grafana/pkg/api/dtos"
 	"github.com/grafana/grafana/pkg/bus"
 	"github.com/grafana/grafana/pkg/infra/metrics"
-	m "github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/util"
 )
 
-func AdminCreateUser(c *m.ReqContext, form dtos.AdminCreateUserForm) {
-	cmd := m.CreateUserCommand{
+func AdminCreateUser(c *models.ReqContext, form dtos.AdminCreateUserForm) {
```

Comment:
That's the note for later - but it feels like more and more that user and every action related to user is smeared across the code base, it seems like it user should be a service...